lib/posix/options/Kconfig.fs

```diff
@@ -15,7 +15,7 @@ if POSIX_FILE_SYSTEM
 config POSIX_FILE_SYSTEM_ALIAS_FSTAT
 	bool
 	help
-	  Select 'y' here and Zephyr will provide an alias for fstat() as _fstat().
+	  When selected via Kconfig, Zephyr will provide an alias for fstat() as _fstat().
```

Comment:
```suggestion
	  When selected, Zephyr will provide an alias for fstat() as _fstat().
```

Reply:
This one is non-user-selectable, so it needs to be done via Kconfig (typically by some libc-specific Kconfig).